x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/helpers.ts

```diff
@@ -69,6 +69,7 @@ export const getExportedRulesCounts = async (blob: Blob): Promise<BulkActionSumm
   const details = await getExportedRulesDetails(blob);
 
   return {
+    skipped: 0,
     succeeded: details.exported_rules_count,
     failed: details.missing_rules_count,
     total: details.exported_rules_count + details.missing_rules_count,
```

Comment:
Curious why we treat `missing_rules_count` as `failed` rather than `skipped`?

Reply:
I'm not sure how this should work. Whar are **missing rules** when attempting to export? I guess we can consider them as `skipped`, but would like to understand this fully before making the change.